seedfarmer/commands/_parameter_commands.py

```diff
@@ -225,3 +228,14 @@ def _module_metatdata(
                 value=parameter_value,
             )
     return None
+
+
+def _resolve_parameter_value_key(valueKey: str) -> str:
+    matches = re.findall(r"<\$(.*?)>", valueKey)
+    for match in matches:
+        _logger.debug(f"Trying to resolve paramter key  {match}")
+        try:
+            valueKey = valueKey.replace("<$" + match + ">", os.environ[match])
```

Comment:
Should we just ask a user to use `$PARAMETER` notation instead of `<$PARAMETER>`? 

`$PARAMETER` seems standard IMO.

Reply:
I need a way to demark where the key ends....what if you added $PARAMETERVALUE and it was supposed to be resolved to SRINIVALUE ?

Reply:
practical use case: ` parameterStore: /idf/$SOMEKEY/somekey` -- I cannot assume that the end of the key is a `/` ...just like `/gibberish$SOMEKEYgibberishagain `. where does the actual string of the key end?

Reply:
Makes sense

Reply:
how about more standard`${PARAM}`?